src/Microsoft.Extensions.Caching.Redis/project.json

```diff
@@ -11,10 +11,21 @@
   },
   "dependencies": {
     "Microsoft.Extensions.Caching.Abstractions": "1.0.0-*",
-    "Microsoft.Extensions.Options": "1.0.0-*",
-    "StackExchange.Redis.StrongName": "1.0.333"
+    "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {}
+    "net451": {
+      "StackExchange.Redis.StrongName": "1.0.488"
+    },
+    "dnxcore50": {
+      "dependencies": {
+        "StackExchange.Redis.StrongName": "1.1.553-alpha"
```

Comment:
Isn't *all of dotnetcore* prerelase? Where is the logical cutoff there? It
seems odd to be concerned with one lib if targeting dotnetcore. Not an
attack: I'm genuinely curious.
On 25 Jan 2016 12:20 a.m., "Ben Adams" <notifications@github.com> wrote:

> In src/Microsoft.Extensions.Caching.Redis/project.json
> <https://github.com/aspnet/Caching/pull/147#discussion_r50646075>:
>
> >    },
> >    "frameworks": {
> > -    "net451": {}
> > +    "net451": {
> > +      "StackExchange.Redis.StrongName": "1.0.488"
> > +    },
> > +    "dnxcore50": {
> > +      "dependencies": {
> > +        "StackExchange.Redis.StrongName": "1.1.553-alpha"
>
> @mgravell <https://github.com/mgravell> or @NickCraver
> <https://github.com/NickCraver> might be able to answer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aspnet/Caching/pull/147/files#r50646075>.
>